spec/features/discover/find_movies_feature_spec.rb

```diff
@@ -2,12 +2,10 @@
 
 RSpec.describe "Discover Movies Page" do
   describe "As an authenticated user" do
-    before :each do
+    it "can search for movies using a keyword", :vcr do
```

Comment:
When we have 2 describes and an it block, the fixtures get nexted in folders I.e. describe/describe/it_fixtures.rb